src/seq/nucleotide.jl

```diff
@@ -103,6 +103,24 @@ function isGC(nt::DNANucleotide)
 end
 
 """
+    isPurine(nt::Nucleotide)
+
+Test if nucleotide is surely a Purine.
+"""
+function isPurine(nt::Nucleotide)
+    return (compatbits(nt) | 0b0101) == 0b0101
```

Comment:
Yes this is what I was trying to achieve, A, G and R, should return true, all others should be false. I believe currently A G and R do return true, and the others do return false - except gaps which also return true.